ampd/src/config.rs

```diff
@@ -131,6 +131,22 @@ mod tests {
             [[handlers]]
             type = 'StellarVerifierSetVerifier'
             cosmwasm_contract = '{}'
+            rpc_url = 'http://localhost:8000'
+
+            [handlers.rpc_timeout]
+            secs = 3
+            nanos = 0
+
+            [[handlers]]
+            type = 'SolanaMsgVerifier'
+            rpc_url = 'http://127.0.0.1'
+            cosmwasm_contract = '{}'
+            max_tx_cache_entries = 6
+
+            [[handlers]]
+            type = 'SolanaVerifierSetVerifier'
+            chain_rpc_url = 'http://127.0.0.1'
```

Comment:
Some entries in this file have `chain_rpc_url` and others `rpc_url`. This one has both,  is that fine? This is a test so I guess this is not super relevant. Also, all other entries use `localhost` instead of `127.0.0.1`. Feel free to ignore this comment, I don't have much context and this just stood out.

Reply:
I think we are prepending `chain` due to [with_prefix!](https://github.com/eigerco/axelar-amplifier/blob/66504ea9f8b12f3a9493f97dae5214a533ab9d4c/ampd/src/handlers/config.rs#L21-L26), thus in tests we can just use both cases, adjusted it to be consistent with rest of the repo and [config](https://github.com/eigerco/axelar-amplifier/blob/66504ea9f8b12f3a9493f97dae5214a533ab9d4c/ampd/src/handlers/config.rs#L66-L75). thanks !

cc168e542b33285321619cef6c0b6bf86be9bf4a and 6499d823dbc02d4852f53466a6c013b3eec05127

Reply:
>adjusted it to be consistent 
Also removed a duplicate entry.